src/resolvers/index.ts

```diff
@@ -70,3 +71,34 @@ export const resolvers = {
     deleteTag,
   },
 };
+
+// Wrap mutations with executeMutation to update the db connection to write
+resolvers.Mutation = Object.keys(resolvers.Mutation).reduce(
```

Comment:
this would be automatically executed without a call as its in global scope right?

Reply:
correct

Reply:
nice, I like that this dries up the code a bit